src/librustc/metadata/common.rs

```diff
@@ -139,14 +139,15 @@ pub enum astencode_tag { // Reserves 0x40 -- 0x5f
     tag_table_adjustments = 0x51,
     tag_table_moves_map = 0x52,
     tag_table_capture_map = 0x53,
-    tag_table_closures = 0x54,
+    tag_table_closure_tys = 0x54,
```

Comment:
I also don't get this -- how could I avoid changing `last_astencode_tag`? I am converting one existing tag into two, and there doesn't seem to be a gap for me to take advantage of...